### PR TITLE
Add type definition file

### PR DESCRIPTION
**Pull request by chang-ke**
#### `Commit Message Summary (CHANGELOG)`

```
commit message body...
```

### `Type`

- [ ] CI
- [x] Fix
- [ ] Perf
- [ ] Docs
- [ ] Test
- [ ] Chore
- [ ] Style
- [ ] Build
- [ ] Feature
- [ ] Refactor

### `SemVer`

- [x] Fix (:label: Patch)
- [ ] Feature (:label: Minor)
- [ ] Breaking Change (:label: Major)

### `Issues`

- Fixes `#1`

### `Checklist`

- [x] Lint and unit tests pass with my changes
- [x] I have added tests that prove my fix is effective/works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes are merged and published in downstream modules

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Type definitions for postcss-load-config 2.1
```
Comment (ai) [inline types/index.d.ts:1]: Do you copy it from `@types/postcss-load-config`. If yes, we need to mention the original project here.

Comment (chang-ke) [reply types/index.d.ts:2]: done

**Review by ai — APPROVED** [review]

**Comment by ai** [comment]
Thanks. I will release it tomorrow (it is too late for a big release for me right now).

File: yarn.lock [hunk 2]
```diff
@@ -1297,8 +1297,8 @@ cosmiconfig@^5.0.0, cosmiconfig@^5.0.7:
 
 cosmiconfig@^7.0.0:
   version "7.0.0"
-  resolved "https://registry.yarnpkg.com/cosmiconfig/-/cosmiconfig-7.0.0.tgz#ef9b44d773959cae63ddecd122de23853b60f8d3"
```
Comment (chang-ke) [inline yarn.lock]: @ai hi. hope to rebase the last two commits into one.

**Comment by ai** [comment]
Thanks. Released in 3.0.1.

**Pull request merged**